File: 1626-can-make-arithmetic-progression-from-sequence/1626-can-make-arithmetic-progression-from-sequence.py
class Solution(object):
    def canMakeArithmeticProgression(self, arr):
        """
        :type arr: List[int]
        :rtype: bool
        """
        arr.sort()
        s = arr[1] - arr[0]
        for i in range(len(arr)-1):
            val = arr[i+1] - arr[i]
            if s != val:
                return False
        return True
        

File: 1626-can-make-arithmetic-progression-from-sequence/1626-can-make-arithmetic-progression-from-sequence.java
class Solution {
    public boolean canMakeArithmeticProgression(int[] arr) {
        Arrays.sort(arr);
        int difference = arr[1] - arr[0];
        for (int i = 1; i < arr.length - 1; i++) {
            int currentDifference = arr[i + 1] - arr[i];
            if (currentDifference != difference) {
                return false; 
            }
        }
        return true;     
    }
}